src/scene_battle_rpg2k.cpp

```diff
@@ -441,10 +441,14 @@ bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase
 				if (battle_message_window->GetLineCount() == 1 && action->IsSecondStartMessage()) {
 					return false;
 				}
-				if (action->GetTarget() &&
-					action->GetTarget()->GetType() == Game_Battler::Type_Enemy) {
-
-					action->PlayAnimation();
+				if (action->GetTarget()) {
+					if (action->GetTarget()->GetType() == Game_Battler::Type_Enemy) {
+						battle_action_wait = GetDelayForLine();
+						action->PlayAnimation();
+					} else if (action->GetTarget()->GetType() == Game_Battler::Type_Ally && action->GetType() == "Skill") {
+						battle_action_wait = GetDelayForLine();
+						action->PlaySoundAnimation();
```

Comment:
afaik I already discussed these "sound" and "cutoff" thinks already with Sormat, so they are fine for me. I find this "GetType()" stuff really verbose, because this is really only used for Skill and only here, so how about adding a new virtual function "IsSkill" instead?

Reply:
I don't like that it's a string compare also. Will look into it.

Reply:
Fixed in #1449